### PR TITLE
Adding Microsoft Auth to Docs

### PR DESCRIPTION
**Pull request by Teeldinho**
##DESCRIPTION
Added auth barrier to the docs.

image

### CODE REVIEW

File: docusaurus.config.ts [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
 import type * as Preset from "@docusaurus/preset-classic";
+// import "dotenv/config";
+require("dotenv").config();
 
 const config: Config = {
   title: "Dark Warehouse",
   tagline: "Intelligent software for efficiently managing traffic in an Omni-Mole warehouse.",
   favicon: "img/favicon.ico",
 
+  customFields: {
```
Comment (itstudes) [inline docusaurus.config.ts:12]: I need you to add an template.env or something and commit to source control. Then please add a section to the README.md file for the project, just so everyone knows how to create the .env file correctly.

File: src/components/ui-components/AccessRequest.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+export default function AccessRequest() {
+  return (
+    <div className="flex items-center justify-center min-h-screen gap-4 min-w-screen bg-stone-900">
+      <div className="flex flex-col items-center w-full max-w-md gap-6 p-8 pt-10 rounded-lg ring-1 ring-stone-700">
+        <ShieldAlert className="text-red-500 size-12" />
+        <div className="flex flex-col items-center w-full gap-2">
+          <h1 className="text-4xl font-semibold text-center text-stone-200">No Access</h1>
+          <div className="pb-4 text-lg text-center text-stone-400">
+            You do not have access to view this page, please contact <span className="font-bold ">info@toastedmarshmallow.co</span> to request access.
```
Comment (itstudes) [inline src/components/ui-components/AccessRequest.tsx:11]: Can you do a mailto: link here?

**Review by itstudes — APPROVED** [review]
This looks great, please just add those files and the README content as per my comment. Then you're welcome to merge it in.

File: src/components/ui-components/AccessRequest.tsx (hunk 2 above)
Comment (Teeldinho) [reply src/components/ui-components/AccessRequest.tsx]: Added.

File: docusaurus.config.ts (hunk 1 above)
Comment (Teeldinho) [reply docusaurus.config.ts:12]: Added.

**Pull request merged**